development/tests/robustness/run.py

```diff
@@ -5,21 +5,13 @@
 
 from soepy.simulate.simulate_python import simulate
 from soepy.test.random_init import random_init
-from soepy.soepy_config import TEST_RESOURCES_DIR
 
 
 def func(maxrt):
     stop = datetime.datetime.now() + maxrt
     while datetime.datetime.now() < stop:
         model_spec_init_dict, random_model_params_df = random_init()
```

Comment:
This will be result in failing regressoin test once commented back in.

Reply:
Fixed